uint8 MaxFilterDistance;
uint8 FilterLifetime;
---